docs/man/pg_activity.1

```diff
@@ -1,4 +1,5 @@
-.\" Automatically generated by Pod::Man 4.14 (Pod::Simple 3.43)
+.\" -*- mode: troff; coding: utf-8 -*-
+.\" Automatically generated by Pod::Man v6.0.2 (Pod::Simple 3.45)
```

Comment:
Can you make a separate commit about the change of `pod2man` version alone, without changing `.pod` file, and then commit the latter (and regenerate `.1`) along with the new feature?